### DISCUSSION TITLE
Firmware 1.4.3

### DISCUSSION THREAD

**Discussion by Meeple85** [Jyers/Marlin#664] [Q&A] [answered]
Bonjour, super boulot merci mais sans trop abuser allez vous faire des pré compilation comme les précédent Firmware . Encore merci

**Comment by Jyers** [comment, marked as answer]
I'm assuming this is referring to Christophe's fork of this firmware. All releases on the main branch have precompiled bins that can be found here: https://github.com/Jyers/Marlin/releases If you need assistance with Christophe's fork I would recommend directing your issue to his repo rather than this one.
